grc/tests/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright 2011-212 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+
+include(GrPython)
+
+if(ENABLE_TESTING)
+    execute_process(
+        COMMAND ${QA_PYTHON_EXECUTABLE} -m pytest --version
```

Comment:
I think we have a module for this.

Reply:
You mean a cmake module that executes pytest? I'm not able to find such module but I'm going to redo this if you tell me where it is.

```
$ git describe
v3.8.0.0-196-g65a8c6c526
$ grep pytest . -R
./grc/tests/test_block_templates.py:import pytest
./grc/tests/test_block_templates.py:    with pytest.raises(TemplateError):
./grc/tests/test_block_templates.py:    with pytest.raises(TemplateError):
./grc/tests/test_expr_utils.py:import pytest
./grc/tests/test_expr_utils.py:    with pytest.raises(RuntimeError):
```

Reply:
I found the `GR_PYTHON_CHECK_MODULE` and `GR_PYTHON_CHECK_MODULE_RAW`. I'm going to rework this

Reply:
`GR_ADD_TEST`?

Reply:
That's what I can find for what uses QA_PYTHON_EXECUTABLE in any capacity in the current GR codebase.

Reply:
GR_PYTHON_CHECK_MODULE is what I meant, sorry, that was a bit cryptic.

Reply:
Done. I've used `GR_PYTHON_CHECK_MODULE_RAW` like in `grc/CMakeLists.txt`